### PR TITLE
13.0 mig product variant sale price

### PR DESCRIPTION
**Pull request by chetanrdhaduk**
Task URL:
https://www.druidoo.io/web#id=1522&action=367&model=project.task&view_type=form&menu_id=237

### CODE REVIEW

File: product_variant_sale_price/__manifest__.py [hunk 1]
```diff
@@ -1,14 +1,15 @@
...
 
 {
     "name": "Product Variant Sale Price",
     "summary": "Allows to write fixed prices in product variants",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Product Management",
     "website": "https://odoo-community.org/",
-    "author": "Tecnativa, " "Odoo Community Association (OCA)",
+    "author": "Tecnativa, " "Odoo Community Association (OCA), Druidoo",
```
Comment (ivantodorovich) [inline product_variant_sale_price/__manifest__.py]: Unfortunately, migrating a module does not make us an author.
Please remove this :)

**Review by ivantodorovich — CHANGES_REQUESTED** [review]

**Review by ivantodorovich — CHANGES_REQUESTED** [review]
After you make this change, please squash this new commit with the Migration to 13.0 one

This looks good :+1: 
I'll test it locally and if it works we push to oca

**Comment by ivantodorovich** [comment]
Another thing to do is squashing all the translation commits into a single one, to clean the history.

> Squash administrative commits (if any) with the previous commit for reducing commit noise. They are named as "[UPD] README.rst", "[UPD] Update $MODULE.pot", "Update translation files" and similar names, and comes from OCA-git-bot, oca-travis or oca-transbot.

**Comment by ivantodorovich** [comment]
In `13.0` we have this "Extra prices" link and it doesn't look good when we hide the price field.

What if we keep the field label `<label for=".."/>` and add an inline comment saying prices are managed at product variant level?

That way we can keep the link, and it's going to look much better

I mean a message similar to this one, with a link to the product variants action:
image





--------





image

**Comment by ivantodorovich** [comment]
Also hide the price on kanban when there are variants:

image

**Comment by ivantodorovich** [comment]
I'm on a variant, and I see the message. Also I'm not able to edit the price there:
image

**Comment by chetanrdhaduk** [comment]
I will remove the message from the variant, As per the functionality in v12 we can not edit the list price directly, We have to update it through "Fix Price" field.

**Pull request closed without merging**